log.go

```diff
@@ -148,6 +164,22 @@ func (this *SrvLog) LogTo(local bool, name, format string, v ...interface{}) {
 		return
 	}
 
+	// append fields to the message
+	fieldCount := len(this.fields)
+	if fieldCount > 0 {
+		keys := make([]string, 0, fieldCount)
+		for key := range this.fields {
+			keys = append(keys, key)
+		}
+		sort.Strings(keys)
+		for index, key := range keys {
+			if index > 0 {
+				msg.Message += ","
+			}
+			msg.Message += fmt.Sprintf(" %s=%+v", key, this.fields[key])
+		}
+	}
+
```

Comment:
Can we eat all of this cost one time by calculating it when WithFields() is called?

Reply:
We could. I guess it depends on whether we think it's more common that fields will be added without a subsequent log in which case doing all the work to format the string will be a waste of time. I could go either way. /shrug